packages/shared_preferences/shared_preferences_foundation/example/ios/RunnerTests/RunnerTests.m

```diff
@@ -0,0 +1,18 @@
+// Copyright 2013 The Flutter Authors. All rights reserved.
+// Use of this source code is governed by a BSD-style license that can be
+// found in the LICENSE file.
+
+@import shared_preferences_foundation;
+@import XCTest;
+
+@interface SharedPreferencesTests : XCTestCase
+@end
+
+@implementation SharedPreferencesTests
+
+- (void)testPlugin {
+  SharedPreferencesPlugin *plugin = [[SharedPreferencesPlugin alloc] init];
```

Comment:
This test is pretty lame, I didn't re-write the better Swift tests back to Objective-C.  Will do that after this lands, to unblock the tree ASAP.

Reply:
Do we have to abandon the Swift tests? I would have thought we could have Swift tests even though the app is Objective-C.

Reply:
I think that does the automagical thing with the Swift runtimes when running an XCTest since Xcode does all kinds of voodoo to get the tests to see non-public APIs?  Relying on this from the example app seems fragile though, it's just luck that we caught it in the storekit example app.  I'll try to think about a better way to holistically test this problem...